autofit/non_linear/search/abstract_search.py

```diff
@@ -530,9 +530,7 @@ class represented by model M and gives a score for their fitness.
                 from pathlib import Path
 
                 search = DynestyStatic(
-                    name=self.name,
-                    unique_tag=self.unique_tag,
-                    path_prefix=Path(self.paths.path_prefix),

```

Comment:
?

Reply:
I'm having to do weird things to get test mode to work with `Nautilus`, but this basically ensures the paths are used correclty.